net/lnet/ut/lnet_ut.c

```diff
@@ -265,15 +265,27 @@ do {								\
 } while (0)
 #define zUT(x) zvUT(x, 0)
 
+#ifndef PAGE_SHIFT
+  #ifdef  CONFIG_X86_64
+    #define PAGE_SHIFT    12
+  #elif defined  CONFIG_AARCH64
+    #define PAGE_SHIFT    16
```

Comment:
Why this has to be 16 here ?

Reply:
Page size on aarch64 is 64KB, hence page shift will be 16

Reply:
@Nalini-Seagate please add a comment about this here also.

Reply:
Fixed